dashboard/config/locales/slides.en.yml

```diff
@@ -1768,3 +1768,149 @@ en:
+      4:
+        image: 'notes/applab_intro_welcome_4.png'
+        text: "The set property block changes the look of the elements on your screen. Like the buttons, labels, or even the screen itself. First you need to decide which element you want to change. If you hover over an element in your app you can see the name or ID here. Then go select that ID from the first drop down."
+      5:
+        image: 'notes/applab_intro_welcome_5.png'
+        text: "Elements have lots of properties you can change like their text color, background color or font size. You can see the full list and choose which property you want to change in the second drop down."
+      6:
+        image: 'notes/applab_intro_welcome_6.png'
+        text: "The last drop down is where you'll write the value you want to use value you want to use. The block will make a suggestion for you. But you can always type in different colors or numbers yourself."
+      7:
+        image: 'notes/applab_intro_welcome_7.png'
+        text: "Once you're done, the block reads a bit like a sentence:'Set button1's background color to green.' Hit run to see the changes you're building in code. If you reset you can drag in more blocks to change other things about your app. Like the text!"
+      8:
+        image: 'notes/applab_intro_welcome_8.png'
+        text: "For this tutorial we'll be working in block mode but App Lab also supports working in text."
+      9:
+        image: 'notes/applab_intro_welcome_9.png'
+        text: "Either way you'll be programming in JavaScript, the language of the web. Each level has a goal, instructions and an image of what you're aiming to create."
+      10:
+        image: 'notes/applab_intro_welcome_10.png'
+        text: "If you're ever stuck click on this picture and it will show you exactly how to complete the level. But see if you can do it on your own first."
+      11:
+        image: 'notes/applab_intro_welcome_11.png'
+        text: "Once you've completed the goal for this level click finish to move on. That's all there is to it! Hit the orange \"continue\" button to go to the next level and start your adventure with an App Lab!"
+    applab_intro_events:
+      1:
+        image: 'notes/applab_intro_events_1.png'
+        text: "Apps are really cool because it's kind of like starting your own business I mean, you get to design everything from the ground up, you get to manage everything, and you're the owner of this piece of work that you just created with code."
+      2:
+        image: 'notes/applab_intro_events_2.png'
+        text: "Now that you can change the look of your app let's make it interactive by responding to events! Events are user actions like clicking a button, scrolling through a menu, or hovering over a picture. Interactive apps need ways to respond to events, like playing a sound when I click this button. To do this in App Lab you need to use a new block called onEvent."
```

Comment:
Not a ship blocker, but we usually add quotes around all block names `"onEvent"`